themes/default/content/blog/iac-recommended-practices-wrapping-up/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Iac Recommended Practices: Wrapping Up"
+date: 2024-03-11
+draft: false
+meta_desc: This wraps up the series on IaC recommended practices, summarizing the previous posts and pointing out areas of future growth for the Zephyr team.
+meta_image: meta.png
+authors:
+    - aaron-kao
+    - christian-nunciato
+    - scott-lowe
+tags:
+    - best-practices
+    - zephyr
+---
+
+Welcome to the final post in our series of articles focused on infrastructure as code (IaC) recommended practices. In this post, we'll wrap up our recommendations for IaC with Pulumi, summarizing the recommendations from previous posts as well as highlighting some areas of potential future growth for the team at Zephyr Archaeotech Emporium---the fictional company at the center of the story throughout this series.<!--more-->
+
+For ease of navigation, here are links to all the blog posts in the series (any entries listed below that aren't linked haven't yet been published; this list will get updated as new posts go live):
```

Comment:
is last part required ? (the any entries listed below that are not linked) since you shortened the series?

Reply:
It is not! I'll remove it (both from this post as well as the other posts.